### PR TITLE
tech: Fix prepare-code script

### PR DESCRIPTION
**Pull request by SpecialAro**
#### Pre-flight Checklist

1. Please remember that if you are logging a bug for some service that has _stopped working_ or is _working incorrectly_, please log the bug [here](https://github.com/ferdium/ferdium-recipes/issues)
2. If you are requesting support for a **new service** in Ferdium, please log it [here](https://github.com/ferdium/ferdium-recipes/pulls)
3. Please remember to read the [self-help documentation](https://github.com/ferdium/ferdium-app#troubleshooting-recipes-self-help) - in case it helps you unblock yourself for issues related to older versions of recipes that were installed on your machine. (These will get automatically upgraded when you upgrade to the newer versions of Ferdium, but to get new recipes between Ferdium releases, this documentation is quite useful.)
4. Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues) for a feature request that matches the one I want to file, without success.

#### Description of Change
Fix prepare-code script - it was failing on reformat-files, extract and compile

#### Motivation and Context
- Fix prepare-code script - it was failing on reformat-files, extract and compile (at least on windows)
- Now we can commit without using `-n`

#### Checklist

- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`npm run prepare-code`)
- [x] `npm test` passes
- [x] I tested/previewed my changes locally

#### Release Notes

tech: Fix prepare-code script

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,12 +28,12 @@
     "test:watch": "jest --watch",
     "lint": "eslint \"{src,test,scripts}/**/*.{js,jsx,ts,tsx}\" --quiet",
     "lint:fix": "npm run lint -- --fix",
-    "extract": "formatjs extract 'src/**/*.{js,jsx,ts,tsx}' --out-file temp.json --flatten --id-interpolation-pattern '[sha512:contenthash:base64:6]' --preserve-whitespace",
-    "compile": "formatjs compile 'temp.json' --out-file src/i18n/locales/en-US.json",
+    "extract": "formatjs extract \"src/**/*.{js,jsx,ts,tsx}\" --out-file temp.json --flatten --id-interpolation-pattern '[sha512:contenthash:base64:6]' --preserve-whitespace",
```
Comment (vraravam) [inline package.json:31]: The old version is working on mac - is it not working on windows with single quotes?

File: package.json [hunk 2]
```diff
@@ -28,12 +28,12 @@
...
+    "compile": "formatjs compile \"temp.json\" --out-file src/i18n/locales/en-US.json",
     "manage-translations": "npm run extract && npm run compile && rimraf temp.json",
     "build": "preval-build-info-cli && gulp build && electron-builder",
     "contributors": "all-contributors",
-    "reformat-files": "./node_modules/.bin/prettier --ignore-path .eslintignore --write --require-pragma \"**/*.{js,jsx,scss}\"",
```
Comment (vraravam) [inline package.json:36]: when I added this in, without the path, `prettier` was not found - maybe that's fixed now, just want to confirm this

**Review by vraravam — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (vraravam) [reply package.json:36]: Please ensure its not picking up a global prettier while verifying this

File: package.json (hunk 1 above)
Comment (SpecialAro) [reply package.json:31]: Yep, it doesn't work on Windows.... Never worked I think.

Just now I got the "head" (insomnia... -.-) to deal with it.

Does this work on macOS this way?

File: package.json (hunk 2 above)
Comment (SpecialAro) [reply package.json:36]: > Please ensure its not picking up a global prettier while verifying this

I'm almost sure I don't have a global `prettier` package installed... Nevertheless let me check

Comment (SpecialAro) [reply package.json:36]: Just checked
image

I don't have any global dependency installed. (As this is a new node installation through nvs, and I frequently run our clean script)

File: package.json (hunk 1 above)
Comment (vraravam) [reply package.json:31]: yes - this works on mac!

**Review by vraravam — APPROVED** [review]
LGTM!

**Pull request merged**